spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -135,6 +135,11 @@ public Distribution requiredDistribution() {
     return requiredDistribution;
   }
 
+  @Override
+  public boolean distributionStrictlyRequired() {
+    return false;
```

Comment:
should we also check `ADAPTIVE_OPTIMIZE_SKEWS_IN_REBALANCE_PARTITIONS_ENABLED` is true as well before disabling this requirement ? otherwise it will be a no-op for `OptimizeSkewInRebalancePartitions`

Reply:
Agree, I was mostly comming from, the point that we are overriding this and setting it to false, in a hope that spark **will** optimize the skew whereas if the above conf is disabled spark will never do the same. I am fine with keeping it as it is.

Reply:
I feel it is better to always return false and leave it us up to Spark. It seems the safest way as Spark may add new configs or logic on when to do that in the future.